### PR TITLE
tests: /api/v1/users/apply endpoint

### LINKED ISSUE #93: tests: /api/v1/users/apply endpoint

### PR DESCRIPTION
**Pull request by hussu010**
closes #93

### CODE REVIEW

File: src/__tests__/user.spec.js [hunk 1]
```diff
@@ -1,18 +1,73 @@
...
+} = require("../constants");
 
-beforeAll(async () => await db.connect());
+beforeAll(async () => {
+  await db.connect();
+  secretKey =
+    "705a3b36b44c2e95eebba870dc6fe5280c5c72783e2fe9449ebb73556faec57e";
+  publicKey =
+    "a5f0766103d33981ca0e90179fbd7ece2c9e76e0ea5d99ebb86cb80ff3c99ab9";
+});
```
Comment (wakawakathedev) [inline src/__tests__/user.spec.js]: dont keep that in git

File: src/__tests__/user.spec.js [hunk 2]
```diff
@@ -1,18 +1,73 @@
...
+
 describe("POST /api/v1/users/create", () => {
   it("should return account number and nonce for valid request", async () => {
     const res = await request(app)
       .post("/api/v1/users/create")
       .send({
         accountNumber:
-          "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+          "a5f0766103d33981ca0e90179fbd7ece2c9e76e0ea5d99ebb86cb80ff3c99ab9",
       })
```
Comment (wakawakathedev) [inline src/__tests__/user.spec.js]: you should use env to run a test-acc creation

**Review by wakawakathedev — COMMENTED** [review]
you should also integrate CI so these tests are run automatically for each commit

File: src/__tests__/user.spec.js (hunk 1 above)
Comment (hussu010) [reply src/__tests__/user.spec.js]: i was actually using the account pairs used https://github.com/LeapChain/DLEAP/blob/main/javascript/test/sign.test.ts
changed it to 00000 to make it obvious

File: src/__tests__/user.spec.js (hunk 2 above)
Comment (hussu010) [reply src/__tests__/user.spec.js]: changed the credentials to account pair for tests

**Review by wakawakathedev — APPROVED** [review]
lgtm

**Pull request merged**